src/components/EditableTagField/index.tsx

```diff
@@ -75,4 +79,5 @@ const FlexColumnContainer = styled.div`
   display: flex;
   flex-direction: column;
   margin-right: 10px;
+  max-width: 70%;
```

Comment:
Can you make sure this doesn't negatively impact the component on the edit spawn host modal

Reply:
I checked, it doesn't.